docs/src/Tools.md

```diff
@@ -204,28 +226,34 @@ The trick, therefore, is to get your host computer to connect to that serial por
 To do that, we a terminal emulator called **screen**.  But first, we have to identify the serial port device that was created when the program started up.
 A simple way to do that is to look in the `/dev/` directory on your computer for a device that includes "usb" in its name:
 
-```
-    ls /dev/*usb*
+```bash
+$ ls /dev/*usb*
 ```
 
-On my machine, this lists two:
+Output on macOS may look like this:
 
+```bash
+/dev/cu.usbmodem14201	/dev/tty.usbmodem14201
 ```
-    /dev/cu.usbmodem14201	/dev/tty.usbmodem14201
+
+On Linux, it may look like this:
+
+```bash
+/dev/cu.usbmodem14201	/dev/tty.usbmodem14201
```

Comment:
This is identical to the Mac version. Consolidate?

Reply:
I hadn't get run the command. I'll update the contents.